### ISSUE TITLE
Discord - Option: Do not log or display embedded data.

### ISSUE THREAD

**Issue by RathaWynter** [miranda-ng/miranda-ng#1901]
My original request for this feature on May 8, 2018 https://github.com/miranda-ng/miranda-ng/issues/1375 was closed due to unrelated fixes and without being resolved. Additionally, requests to have it opened again have gone unnoticed and or unanswered for many months. I would like to have this issue reconsidered both because support for embedded data is not part of the core plugin set for Miranda, and because adding text to the log that was not sent as part of the original message seems like it should be optional and not the default behavior of a messenger. (But I would be happy just to have the option to not show or log it.)

Relevant for Miranda 0.95.11#21617 and previous versions of the Discord plugin.

### The Issue:
Linking any content with embedded data (such as Youtube videos or Steam store pages) over the Discord protocol creates significant amounts of garbage / clutter in the chat window and in the history when viewed with TabSRMM (including history++ and the default history viewer.) This embed data is often many times the size of the original message, **and includes data that was not sent as part of the original message.**

### An Example:
Sending the following message in "Original Message" results in the following output:

**Original Message** (43 Characters, 1 line)
`https://www.youtube.com/watch?v=lbjRlxvKLBc`

**Displayed / Logged Message** (~350 characters, 7+ lines)
```
Ratha  Today
2:19 PM https://www.youtube.com/watch?v=lbjRlxvKLBc
-----------------
Embed: https://www.youtube.com/watch?v=lbjRlxvKLBc
	YouTube video
	Want to see more WowCraft cartoons? Support: http://www.patreon.com/carbotanimations SHIRTS: http://gear.blizzard.com/index.php/default/starcrafts/ Follow on...
Preview: https://i.ytimg.com/vi/lbjRlxvKLBc/hqdefault.jpg
```

### The Request:
Would like an option for the Discord protocol to not display or log any of this additional data: "Disable embedded data logging"

Simply put: Only display and log the original text message that was sent to the user. Do not include any additional retrieved embedded text or data.

**Comment by georgehazan** [comment]
protocol isn't in charge of logging network data, the NetLib/kernel does it automatically for all protocols, so there's no chance to introduce such option

**Comment by RathaWynter** [comment]
Im not an expert here, but it seems to me like some of the lines between 180-225 https://github.com/miranda-ng/miranda-ng/blob/da2c4abfc249d194d36b4bdc5dca1b711a9685f1/protocols/Discord/src/utils.cpp in the source code for the Discord plugin are what is 'adding' all of the embed text and cluttering up the logs.

Preparing the delimiter, appending "\n-----------------" adding the description, adding the thumbnail. Seems like most of that could be skipped with an option in the discord plugin.

**Comment by georgehazan** [comment]
this code analyzes json received and appends its contents to the event's body, but the fact is that all that json is already logged, thus creating an impression that these lines are logged
